.travis.yml

```diff
@@ -12,6 +12,11 @@ php:
     - 7.1
 
 matrix:
+    exclude:
+        - php: 5.6
+          env: DEPENDENCIES='beta' SYMFONY_VERSION='4.0.*' STABLE=beta
```

Comment:
You can remove beta, Symfony 4 is stable now

Reply:
@garak DONE